packages/extension-browser/src/devtools/views/app.base.css

```diff
@@ -5,6 +5,7 @@
     --base-color: #000;
 
     --code-bg: var(--base-bg);
+    --popup-bg: #efefef;
```

Comment:
Should we call this `--notification-bg` for consistency with how the component is named?